tests/mock_network/utils.rs

```diff
@@ -628,11 +629,12 @@ pub fn nodes_with_prefix_mut<'a>(
 }
 
 pub fn verify_section_invariants_for_node(node: &TestNode, elder_size: usize) {
-    let our_prefix = unwrap!(
-        node.inner.our_prefix(),
-        "{} does not have prefix",
-        node.inner
-    );
+    let our_prefix = match node.inner.our_prefix() {
+        Some(pfx) => pfx,
+        None => {
+            return;
```

Comment:
Why is this OK now? After `poll_and_resend`, every node should be approved (infant, adult or elder) so it should have prefix. Or am I wrong?

Reply:
Fixed with https://github.com/maidsafe/routing/pull/1959